a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -206,12 +206,17 @@ def get_delete_load_balancer_flow(self, lb, deleteCompute):
             requires=(constants.LOADBALANCER, a10constants.VTHUNDER),
             provides=a10constants.STATUS))
 
-        # delete_LB_flow.add(listeners_delete)
-        # delete_LB_flow.add(network_tasks.UnplugVIP(
-        #    requires=constants.LOADBALANCER))
-        # delete_LB_flow.add(network_tasks.DeallocateVIP(
-        #    requires=constants.LOADBALANCER))
+        # TODO: delete flow for flat network
+        network_type = 'vlan'
         if deleteCompute:
+            if network_type == 'vlan':
+                delete_LB_flow.add(a10_network_tasks.DeallocateTrunk(
+                    requires=constants.LOADBALANCER,
+                    provides=a10constants.TRUNK))
+                delete_LB_flow.add(a10_network_tasks.UnplugVIP(
```

Comment:
This code should be taken from upstream. If present in our network tasks then it should be deleted.